tests/cli_image_operation_args.rs

```diff
@@ -0,0 +1,138 @@
+#[macro_use]
+pub mod common;
+
+// fixme: add additional integration tests
```

Comment:
This is a 'nice to have'. Most of this is already quite ok tested by their own component tests.